frontend/src/global_styles/content/_pagination.sass

```diff
@@ -67,8 +67,9 @@ $pagination--font-size: 0.8125rem
       min-width: 25px
 
     &_current
-      background: #ffffff
-      border: 1px solid #ffffff
+      background: var(--bgColor-emphasis)
+      border: 1px solid var(--borderColor-emphasis)
+      color: var(--fgColor-onEmphasis)
```

Comment:
These are not the correct colors as it looks off in the normal and dark mode

<img width="264" alt="Bildschirmfoto 2024-06-12 um 07 17 05" src="https://github.com/opf/openproject/assets/7457313/23af66e0-6864-4b06-8f62-fc3acad286e8">
<img width="396" alt="Bildschirmfoto 2024-06-12 um 07 35 01" src="https://github.com/opf/openproject/assets/7457313/a94dca89-0a1b-4592-866f-18e353e54ff9">